02.calendar/calender.rb

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env ruby
+#
+# ABOUT
+# ・コマンドラインで実行できるカレンダー表示スクリプトです
+# Usage
+# ・月と年を以下オプションの引数で設定し、該当年月のカレンダーを表示させる
+# -m : 1,2,3,4,5,6,7,8,9,10,11,12
+# -y : yyyy
+
+require 'date'
+require 'optparse'
+
+# デフォルトの表示年月を、今月に設定
+print_date = Date.today
+argument_month = print_date.month
+argument_year = print_date.year
```

Comment:
下の方のコードと合わせてこういうふうに書くとスマートかもです〜

```suggestion
argument_month = (params['m'] || print_date.month).to_i
argument_year = (params['y'] || print_date.year).to_i
```